### DISCUSSION TITLE
det_db_unclip_ratio的值，能否在模型加载之后继续修改？

### DISCUSSION THREAD

**Discussion by ZhuangPeiyu** [PaddlePaddle/PaddleOCR#13117] [Q&A]
使用PaddleOCR起一个服务，后续不同场景调用同个服务的时候，不同的场景需要设置不同的det_db_unclip_ratio的值，但是Paddle OCR貌似在启动的时候指定det_db_unclip_ratio的值之后，这个参数就不能再进行修改了？

**Comment by SWHL** [comment]
这个需要自己修改一下源码了，后期也许会开放出来。
